### ISSUE TITLE
grunt-nsp and nsp CLI tool has different output

### ISSUE THREAD

**Issue by pdehaan** [requiresafe/grunt-requiresafe#9]
Possibly/probably related to https://github.com/nodesecurity/grunt-nsp/issues/4, but I just noticed that we currently don't display the patched version when calling through `$ grunt nsp`:

``` sh
$ grunt nsp --force
Running "nsp" task
┌───────────────┬──────────────────────────────────────────────────────────┐
│               │ Regular Expression Denial of Service                     │
├───────────────┼──────────────────────────────────────────────────────────┤
│ Name          │ uglify-js                                                │
├───────────────┼──────────────────────────────────────────────────────────┤
│ Version       │ 2.3.6                                                    │
├───────────────┼──────────────────────────────────────────────────────────┤
│ Path          │ bower,handlebars,uglify-js                               │
├───────────────┼──────────────────────────────────────────────────────────┤
│ More Info     │ https://nodesecurity.io/advisories/48                    │
└───────────────┴──────────────────────────────────────────────────────────┘
```

... and here's the output from `$ nsp check`:

``` sh
$ nsp check --warn-only
(+) 16 vulnerabilities found
┌───────────────┬──────────────────────────────────────────────────────────┐
│               │ Regular Expression Denial of Service                     │
├───────────────┼──────────────────────────────────────────────────────────┤
│ Name          │ uglify-js                                                │
├───────────────┼──────────────────────────────────────────────────────────┤
│ Installed     │ 2.3.6                                                    │
├───────────────┼──────────────────────────────────────────────────────────┤
│ Vulnerable    │ All                                                      │
├───────────────┼──────────────────────────────────────────────────────────┤
│ Patched       │ None                                                     │
├───────────────┼──────────────────────────────────────────────────────────┤
│ Path          │ bower > handlebars > uglify-js                           │
├───────────────┼──────────────────────────────────────────────────────────┤
│ More Info     │ https://nodesecurity.io/advisories/48                    │
└───────────────┴──────────────────────────────────────────────────────────┘
```

**Comment by evilpacket** [comment]
Yup I'll be fixing this when I fix the formatter request as well.

**Comment by evilpacket** [comment]
fixed by grunt-nsp@2.1.0
